Blocks.lean

```diff
@@ -0,0 +1,648 @@
+/-
```

Comment:
@AntoineChambert-Loir I suppose that this shouldn't be part of this PR, right?

Reply:
yes, sorry.

Reply:
in fact it should have been, but the next two files shouldn't…